composer.json

```diff
@@ -37,10 +37,9 @@
         "symfony/security-bundle": "^2.8 || ^3.2 || ^4.0",
         "symfony/templating": "^2.8 || ^3.2 || ^4.0",
         "symfony/translation": "^2.8 || ^3.2 || ^4.0",
-        "symfony/twig-bridge": "^2.8 || ^3.2 || ^4.0",
```

Comment:
Are you sure it does not depend also on the bridge: https://github.com/sonata-project/SonataAdminBundle/blob/d6ffaef34cf6e0b08a12478478d5105eec592b50/src/Controller/HelperController.php#L19-L22 ?

Reply:
@greg0ire `symfony/twig-bundle` depends on `symfony/twig-bridge` and `twig/twig`. So, with one new dependency we can remove two old

See https://github.com/symfony/twig-bundle/blob/master/composer.json

Reply:
No we can't: the bundle depends on the bridge and on twig because it references them in its code. If, for some reason, it no longers needs the bridge for instance, they can remove the dependency from the bundle and it won't be considered a BC-break. Also, if they suddenly allow twig-bridge 5 but we are not yet compatible with it, there will be bugs. Please add this back.

Reply:
@greg0ire i understood, thank you for explanation. I've just added `symfony/twig-bundle` dependency